src/main/java/org/cloudfoundry/identity/uaa/api/group/UaaGroupOperations.java

```diff
@@ -81,7 +85,7 @@
 	 * @return The page of groups.
 	 * @see org.cloudfoundry.identity.uaa.api.common.model.expr.FilterRequestBuilder
 	 */
-	public PagedResult<UaaGroup> getGroups(FilterRequest request);
+	public ScimGroups getGroups(FilterRequest request);
```

Comment:
No, but I did find a solution last night that will solve the issue using `SearchResults<T>`. I have to subclass `SearchResults<T>` for right now because there's no default constructor for it in 2.1.0 (though it _is_ in the `develop` branch).